dtfs-central-api/src/helpers/amendments/get-effective-facility-value-amendment.ts

```diff
@@ -0,0 +1,26 @@
+import { TfmFacility } from '@ukef/dtfs2-common';
+import { filterAndSortCompletedEffectiveAmendments } from './filter-and-sort-completed-effective-amendments';
+
+/**
+ * Get the completed amendment facility value that is effective
+ * on the effective at date if there are any
+ * @param tfmFacility - The tfm facility
+ * @param effectiveAtDate - The date to find the effective amendment for
+ * @returns The latest completed amendment cover end date
+ */
+export const getEffectiveFacilityValueAmendment = (tfmFacility: TfmFacility, effectiveAtDate: Date): number | null => {
```

Comment:
does this still make sense as getLatestEffectiveFacilityValueAmendment? Ditto for the other function

Reply:
ah yeah that does make sense tbf, i was thinking too much about the meaning of effective date rather than just the fact that effective is it's own meaning that works here :D getting in my own head writing this comment